tests/unit_tests/test_material.py

```diff
@@ -477,16 +477,20 @@ def test_activity_of_stable():
     m1.add_element("Fe", 1)
     m1.set_density('g/cm3', 1)
     m1.volume = 1
-    assert m1.activity == 0
+    assert m1.get_activity() == 0
```

Comment:
Let's combine these tests into `test_get_activity()` Also set the density to a float like 1.5 and check the activity per gram and per cc first before setting the `volume` property then check total activity.

Reply:
Is that any better, I've added some extra asserts to the stable nuclide test.

Reply:
Can I just check which unit tests do you want combining, we currently have 4
```
test_get_activity_of_stable_nuclides
test_get_activity_of_tritium
test_get_activity_of_metastable
test_get_activity_of_tritium_nuclides
```

Reply:
Yes, I think those 4 should be combined into a single unit test named `test_get_activity` after the function itself, make sense?